.github/workflows/pull-request.yaml

```diff
@@ -26,38 +26,12 @@ jobs:
       with:
         egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
 
-    - uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28
```

Comment:
@jenstroeger by removing the `checkout` step here, we get a [warning](https://github.com/jenstroeger/python-package-template/runs/6717525442?check_suite_focus=true#step:4:9): `rules module not found` because it can't read [commitlint.rules.js](https://github.com/jenstroeger/python-package-template/blob/main/commitlint.rules.js).

We could as well remove the `commitlint.rules.js` file if we are happy with this warning?

Reply:
Hm, looking at the [default settings](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional) the `commitlint.rules.js` we have changes [`body-max-line-length`](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional#body-max-line-length) from 100 to 200. Also, unfortunately, the [Conventional Commit Action](https://github.com/CondeNast/conventional-pull-request-action) doesn’t support passing these config values through the command line either.

So I guess yes, we might as well remove that file…

Reply:
see [3d6a52d](https://github.com/jenstroeger/python-package-template/pull/211/commits/3d6a52d212bae6c362219cdf2a8a7193ee3248a6).